tests/models/dac/test_modeling_dac.py

```diff
@@ -681,10 +681,10 @@ def test_integration_batch_44khz(self):
         expected_rmse = 0.001
 
         expected_encoder_sums_dict = {
-            "loss": 51.8182,
-            "quantized_representation": 778.2920,
-            "audio_codes": 778.2920,
-            "projected_latents": -3844.6055,
+            "loss": 51.8466,
+            "quantized_representation": 595.9933,
+            "audio_codes": 2121648,
```

Comment:
These are very large differences 👀  Do we expect more variation for e.g. the 44khz case?

Reply:
We are comparing the `sum` over values in each tensor, which I guess propagates errors and explain why we result with such a high difference. 

I switched to using `mean` instead, this seems to reduce the observed differences.